build.gradle

```diff
@@ -115,6 +117,18 @@ allprojects {
 
 apply from: script("ide")
 
+allprojects {
```

Comment:
this shouldn't be here, it is unrelated to the pr

Reply:
The old Asciidoctor plugin added repositories to the build, the newer versions do not.
That's why I moved the repositories Block from the `subprojects` closure to the `allprojects` closure.